src/main/java/com/minecolonies/coremod/colony/jobs/JobBaker.java

```diff
@@ -0,0 +1,121 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.coremod.client.render.RenderBipedCitizen;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.baker.EntityAIWorkBaker;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.SoundEvent;
+import org.jetbrains.annotations.NotNull;
+
+
+/**
+ * The fisherman's job class.
+ * implements some useful things for him.
+ */
+public class JobBaker extends AbstractJob
+{
+    /**
+     * Initializes the job class.
+     *
+     * @param entity The entity which will use this job class.
+     */
+    public JobBaker(final CitizenData entity)
+    {
+        super(entity);
+    }
+
+    /**
+     * Restore the Job from an NBTTagCompound.
+     *
+     * @param compound NBTTagCompound containing saved Job data.
+     */
+    @Override
+    public void readFromNBT(@NotNull final NBTTagCompound compound)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1185)